mibidata/tests/test_mibi_image.py

```diff
@@ -138,6 +138,23 @@ def test_check_folder_without_fov_id(self):
         self.assertEqual(image.fov_id, 'Point1')
         self.assertEqual(image.folder, 'Point1/RowNumber0/Depth_Profile0')
 
+    def test_set_fov_id_no_folder(self):
+        image = mi.MibiImage(TEST_DATA, TUPLE_LABELS, fov_id='FOV1')
+        image.set_fov_id('FOV2')
+        self.assertEqual(image.fov_id, 'FOV2')
+        self.assertEqual(image.folder, 'FOV2')
+
+    def test_set_fov_id_bad_folder(self):
+        image = mi.MibiImage(TEST_DATA, TUPLE_LABELS, fov_id='FOV1')
+        with self.assertRaises(ValueError):
+            image.set_fov_id('FOV2', 'FOV3')
+
+    def test_set_fov_id_with_folder(self):
+        image = mi.MibiImage(TEST_DATA, TUPLE_LABELS,
+                             folder='Point1/RowNumber0/Depth_Profile0')
+        image.set_fov_id('Point2', 'Point2/RowNumber0/Depth_Profile0')
```

Comment:
You can add one more test here where you do `image.set_fov_id('Point1')` without the `folder`argument, and check if folder stays at `'Point1/RowNumber0/Depth_Profile0'`

Reply:
I claim that is the same as the first test.